frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/utils.ts

```diff
@@ -143,19 +143,23 @@ export function useCheckCollectionFormatsPopulated({
   dataset: Dataset;
   datasetUploadStatus: DatasetUploadStatus;
 }): void {
-  let numOfConvertedFormats = 0;
+  let numOfProcessedFormats = 0;
 
   for (const key of CONVERSION_STATUS_FORMAT_KEYS) {
-    if (datasetUploadStatus[key] !== CONVERSION_STATUS.CONVERTED) continue;
-
-    numOfConvertedFormats += 1;
+    if (
+      [CONVERSION_STATUS.CONVERTED, CONVERSION_STATUS.SKIPPED].some(
+        (status) => status === datasetUploadStatus[key]
+      )
+    ) {
+      numOfProcessedFormats += 1;
+    }
   }
 
   const numOfAvailableFormats = dataset.dataset_assets.length;
 
   useCheckCollection({
     invalidateCollectionQuery,
-    shouldFetch: numOfConvertedFormats > numOfAvailableFormats,
+    shouldFetch: numOfProcessedFormats > numOfAvailableFormats,
```

Comment:
I haven't tried running this, but since we're introducing the possibility that the number of process formats(converted or skipped) will be more than available.  There will be cases where `shouldFetch` will be true when it shouldn't be?